src/app/button-group/angular/hide-show-overflow-toggle/hide-show-overflow-toggle.html

```diff
@@ -1,3 +1,9 @@
+<!--
```

Comment:
Do we really need to add these? When its bundled templates are moved into the JS anyways. This is also not shipped code, just our sample app.

Reply:
IntelliJ added them when I processed the `/src` directory. I can see if I can change the way it adds multi-line comments.